pyiron_atomistics/sphinx/interactive.py

```diff
@@ -51,8 +51,8 @@ def structure(self, structure):
                 element_lst=structure.get_species_symbols().tolist()
             )
 
-    def get_structure(self, iteration_step=-1, wrap_atoms=True):
-        return GenericInteractive.get_structure(
+    def _get_structure(self, iteration_step=-1, wrap_atoms=True):
```

Comment:
Yeah, in contrast I definitely see the need for the explicit super call here since the `_get_structure` method differs depending whether you wind up at `GenericDFTJob` or `AtomisticGenericJob`.